.github/workflows/NightlyBaselineProfiles.yaml

```diff
@@ -0,0 +1,40 @@
+name: NightlyBaselineProfiles
+
+on:
+  schedule:
+    - cron:  '42 4 * * *'
+
+jobs:
+  baseline_profiles:
+    name: "Generate Baseline Profiles"
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
+
+    timeout-minutes: 60
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Validate Gradle Wrapper
+        uses: gradle/wrapper-validation-action@v1
+
+      - name: Copy CI gradle.properties
+        run: mkdir -p ~/.gradle ; cp .github/ci-gradle.properties ~/.gradle/gradle.properties
+
+      - name: Set up JDK 17
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'zulu'
+          java-version: 17
+
+      # Replace task exclusions with `-Pandroidx.baselineprofile.skipgeneration` when
```

Comment:
How come you're using it in the other yaml file and not here then?

Also: which task exclusions?

Reply:
Copy-paste issue. This needs to be removed